DuckDuckGo/PreserveLoginsSettingsViewController.swift

```diff
@@ -47,23 +48,28 @@ class PreserveLoginsSettingsViewController: UITableViewController {
     @IBAction func startEditing() {
         navigationItem.setHidesBackButton(true, animated: true)
         navigationItem.setRightBarButton(doneButton, animated: true)
+
+        tableView.setEditing(false, animated: true)
+        tableView.setEditing(true, animated: true)
         
-        // Fix glitch happening when there's cell that is already in the editing state (swiped to reveal delete button) and user presses 'Edit'.
```

Comment:
@miasma13 What do you think about restoring this comment for the two lines above? It's not obvious what the intent is for those two lines without the explanation. 🙂

Reply:
Good idea, will do.